From eee3a2753fdb5993354ebc7525f6fee9152b38fc Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Fri, 5 Jun 2015 15:09:18 -0500
Subject: [PATCH 0741/5242] MLK-11061: ARM:dts: enable ddr quad mode for QSPI
 on 6UL ARM2 and 7D ARM2

commit  a215cb3b655c89b4a78cd2a1c2b631f3ef1e8d1d from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable DDR Quad mode for QSPI on 6UL ARM2 and 7D ARM2 board to improve
the performance.

Signed-off-by: Han Xu <b45815@freescale.com>
(cherry picked and merge from commit ba6c2f00b5b000996bf8acbbd62c30d8f8f61182)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts
index 4f2eb33..ede6df2 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-qspi.dts
@@ -50,6 +50,7 @@
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
 		spi-max-frequency = <29000000>;
+		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <0>;
 	};
 
@@ -58,6 +59,7 @@
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
 		spi-max-frequency = <29000000>;
+		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <1>;
 	};
 
@@ -66,6 +68,7 @@
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
 		spi-max-frequency = <29000000>;
+		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <2>;
 	};
 
@@ -74,6 +77,7 @@
 		#size-cells = <1>;
 		compatible = "micron,n25q256a";
 		spi-max-frequency = <29000000>;
+		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <3>;
 	};
 };
-- 
1.7.9.5

